pkg/adapter/bitbucket/adapter.go

```diff
@@ -0,0 +1,180 @@
+/*
+Copyright 2019 The Knative Authors
```

Comment:
I'd think this source goes to the `contrib` folder

Reply:
> I'd think this source goes to the `contrib` folder

I agree.

Reply:
Sounds good! Will move it there and let you guys know.
Thanks!